specification/riskiq/data-plane/Microsoft.Easm/preview/2022-04-01-preview/easm.json

```diff
@@ -0,0 +1,5041 @@
+        },
+        "registrantContacts": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrarCreatedAt": {
+          "items": {
+            "$ref": "#/definitions/ObservedLong"
+          },
+          "type": "array"
+        },
+        "registrarNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrarUpdatedAt": {
+          "items": {
+            "$ref": "#/definitions/ObservedLong"
+          },
+          "type": "array"
+        },
+        "registries": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "sources": {
+          "items": {
+            "$ref": "#/definitions/Source"
+          },
+          "type": "array"
+        },
+        "technicalContacts": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalOrgs": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AssetId": {
+      "description": "The list of excludes used for the disco group runs, aka assets to exclude from the discovery algorithm.",
+      "properties": {
+        "displayName": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "kind": {
+          "enum": [
+            "as",
+            "contact",
+            "domain",
+            "host",
+            "ipAddress",
+            "ipBlock",
+            "page",
+            "sslCert"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "assetIdKind"
+          }
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "AssetLabelRequest": {
+      "properties": {
+        "transfers": {
+          "enum": [
+            "AS",
+            "CONTACT",
+            "DOMAIN",
+            "HOST",
+            "IP_ADDRESS",
+            "IP_BLOCK",
+            "PAGE",
+            "SSL_CERT"
```

Comment:
We recommend enum values are camelCase, as they were prior.

Reply:
Yup caught and fixed this with the 2022-09-01-preview version.  We will be opening up a PR for that merge after this one is merged down. ;-)